README.md

```diff
@@ -90,6 +90,7 @@ Example of the CDC position:
 | `tabletType`     | Specifies a VTGate tablet type.                                                                                                                                              | false    | `primary`                                                 |
 | `columns`        | Comma separated list of column names that should be included in each Record's payload.<br />If the field is not empty it must contain a value of the `orderingColumn` field. | false    | all columns                                               |
 | `batchSize`      | Size of rows batch. Min is `1` and max is `100000`.                                                                                                                          | false    | `1000`                                                    |
+| `retries`        | Specifies the grpc retries to vitess.                                                                                                                                        | false    | `3`                                                       |
```

Comment:
_The number of reconnect retries the connector will make before giving up if a connection goes down._

Also, consider using this phrase in the godoc comments, spec, and destination.